src/names.h

```diff
@@ -23,7 +23,7 @@
 NETPLAN_INTERNAL const char*
 netplan_backend_name(NetplanBackend val);
 
-const char*
+NETPLAN_INTERNAL const char*
 netplan_def_type_name(NetplanDefType val);
```

Comment:
If we were to keep it internal we want a `_`. However, I'd vote for having it part of the public API, as it makes life much easier for any consumer that's not speaking C natively, they can get human-readable strings instead of an opaque enum value that they need to write brittle matching code for.

Reply:
This discussion makes me think we have a `name`-shaped hole in our spec ;)

Reply:
Indeed. But those functions are still `NETPLAN_INTERNAL`, so I think we can come up with a more appropriate naming scheme for `*_name` functions in a follow-up PR where we make things align with the API spec.